arangodb-net-standard/GraphApi/GraphApiClient.cs

```diff
@@ -190,5 +191,39 @@ public async Task<PostGraphEdgeResponse> PostGraphEdgeDefinitionAsync(string gra
                 throw await GetApiErrorException(response);
             }
         }
+
+        /// <summary>
+        /// Adds a vertex to the given collection.
+        /// POST/_api/gharial/{graph}/vertex/{collection}
+        /// </summary>
+        /// <typeparam name="T"></typeparam>
+        /// <param name="graphName"></param>
+        /// <param name="collectionName"></param>
+        /// <param name="body"></param>
+        /// <param name="query"></param>
+        /// <returns></returns>
+        public async Task<PostVertexResponse> PostVertexAsync<T>(
+            string graphName,
+            string collectionName,
+            T body,
+            PostVertexQuery query = null)
+        {
+            string uri = _graphApiPath + '/' + WebUtility.UrlEncode(graphName) +
+                "/vertex/" + WebUtility.UrlEncode(collectionName);
+            if (query != null)
+            {
+                uri += "?" + query.ToQueryString();
+            }
+            StringContent content = GetStringContent(body, true, true);
```

Comment:
This should be `body, false, false` from what I understand, otherwise we are going to mess with the properties that the user actually wants to create in the vertex.